javascript/packages/cli/src/cli.ts

```diff
@@ -85,7 +85,12 @@ program
       ["podman", "kubernetes", "native"],
     ),
   )
-
+  .addOption(
+    new Option(
+      "-l, --logType <logType>",
+      "Type of logging - defaults to 'table'",
+    ),
```

Comment:
```suggestion
    ).choices(["table", "text", "silent"]),
```
We can set the available choices in the `cli`.